Src/Test.UnitTests.Sarif.PatternMatcher.Cli/ProgramTests.cs

```diff
@@ -22,6 +23,26 @@ namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Cli
 {
     public class ProgramTests
     {
+        [Fact]
+        public void Program_ResponseFileWorksWithNoExplicitlyAllocatedFileSystem()
+        {
+            string responseFilePath = $"@{Guid.NewGuid()}.rsp";
+
+            string[] args = new[] { responseFilePath };
+            int result = Program.Main(args);
+            result.Should().Be(1);
+
+            // These conditions indicate that the response file argument was
+            // successfully processed by the command-line argument parsers 
+            // and the response file was not found. This closes a very narrow
+            // code path in tests where an uninstantiated FileSystem object
+            // causes a null dereference. All other tests for this area provide
+            // a non-null file system object instance.
+            Program.RuntimeException.Should().NotBeNull();
+            Program.RuntimeException.GetType().Should().Be(typeof(FileNotFoundException));
```

Comment:
>FileNotFoundException [](http://example.com/codeflow?start=66&length=21)

not sure why but when we run this using the commandline, the tests don't pass.

Maybe debug vs release?

Reply:
We saw that the real issue was how the order of things were happening. To enforce that the behavior will be always the same, michael pushed a change enforcing FileSystem = null.